### PR TITLE
[PEPC-Boost][14] Goerli state interference checks

### PR DESCRIPTION
**Pull request by bharath-123**
## 📝 Summary

Goerli state interference unit tests

### CODE REVIEW

File: services/api/pepc_boost_test.go [hunk 1]
```diff
@@ -141,6 +141,30 @@ func GetCustomDevnetTracingRelatedTestData(t *testing.T) (*gethtypes.Transaction
 	return validWethDaiTx, validWethDaiTxTrace, invalidWethDaiTx, invalidWethDaiTrace
 }
 
+func GetGoerliTracingRelatedTestData(t *testing.T) (*gethtypes.Transaction, *common.CallTrace, *gethtypes.Transaction, *common.CallTrace) {
+	validEthUsdcTxContents := common.LoadFileContents(t, "../../testdata/traces/goerli/valid_eth_usdc_tx.json")
```
Comment (michaelneuder) [inline services/api/pepc_boost_test.go:145]: t.Helper()

File: services/api/pepc_boost_test.go [hunk 2]
```diff
@@ -206,6 +247,110 @@ func TestIsTxWEthDaiSwap(t *testing.T) {
...
+		description    string
+		callTrace      common.CallTrace
+		isTraceCorrect bool
+		requiredError  string
+	}{
+		{
+			description:    "valid trace",
+			callTrace:      *ethUsdcTrace,
+			isTraceCorrect: true,
+			requiredError:  "",
```
Comment (michaelneuder) [inline services/api/pepc_boost_test.go:324]: seems like `requiredError` isnt used in any of the cases? worth removing if so

File: services/api/pepc_boost_test.go [hunk 3]
```diff
@@ -633,11 +823,96 @@ func TestSubmitTobTxsInSequence(t *testing.T) {
...
+					GasPrice: big.NewInt(5),
+					Gas:      12,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(10),
+					Data:     []byte(""),
+				}),
+			},
+			firstTobTxsTraces:  validEthUsdcTxTrace,
+			secondTobTxsTraces: validEthUsdcTxTrace,
+			network:            common.EthNetworkGoerli,
```
Comment (michaelneuder) [inline services/api/pepc_boost_test.go:909]: why do these need to be specific to goerli ? isnt this generic enough for all networks?

**Review by michaelneuder — APPROVED** [review]
trying my best lol.

File: services/api/pepc_boost_test.go (hunk 2 above)
Comment (bharath-123) [reply services/api/pepc_boost_test.go:324]: true will remove!

File: services/api/pepc_boost_test.go (hunk 3 above)
Comment (bharath-123) [reply services/api/pepc_boost_test.go:909]: The smart contract addresses change for each n/w dude. Thats I making these checks n/w specific. Based on the n/w type we fetch the smart contract addresses for which we check the `To` address of the callTrace call etc.

**Pull request merged**